### PR TITLE
chore(deps): Bump Prettier to 3.1.0, content changes

### PR DESCRIPTION
**Pull request by bsmth**
This PR includes some fixes in content required to get tests passing while bumping this dep.

Changes:

* [Ternaries are nested](https://prettier.io/blog/2023/11/13/3.1.0.html#add-indentation-back-to-nested-ternaries-9559httpsgithubcomprettierprettierpull9559-by-rattrayalexhttpsgithubcomrattrayalex)
* [Breaking in assignments](https://prettier.io/blog/2023/11/13/3.1.0.html#improve-formatting-for-assignment-its-left-can-break-15547httpsgithubcomprettierprettierpull15547-by-sosukesuzukihttpsgithubcomsosukesuzuki)
* [Marquee element](https://prettier.io/blog/2023/11/13/3.1.0.html#fix-formatting-of-menu-and-marquee-elements-15334httpsgithubcomprettierprettierpull15334-by-fiskerhttpsgithubcomfisker)
* [Unary expr on left of instanceof](https://prettier.io/blog/2023/11/13/3.1.0.html#disambiguate-unary-expressions-on-left-hand-side-of-instanceof-and-in-15468httpsgithubcomprettierprettierpull15468-by-lucacasonatohttpsgithubcomlucacasonato)

Tested on this branch:

```bash
 bsmth npm i -g prettier@3.1.0
...
 bsmth prettier -v
3.1.0
 bsmth prettier -c "**/*.md"
Checking formatting...
All matched files use Prettier code style!
```

### CODE REVIEW

File: files/en-us/web/javascript/reference/operators/instanceof/index.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ if (!(mycar instanceof Car)) {
 This is really different from:
 
 ```js example-bad
-if (!mycar instanceof Car) {
+if ((!mycar) instanceof Car) {
```
Comment (bsmth) [inline files/en-us/web/javascript/reference/operators/instanceof/index.md:218]: Maybe we want to ignore / nolint this block

Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/operators/instanceof/index.md:218]: Right, please add holing

**Review by teoli2003 — APPROVED** [review]
👍 Happy to see MDN being so easy to adapt for a "medium" release like 3.1.0!

**Comment by teoli2003** [comment]
@bsmth I think the failing test of Markdownlint is unrelated, but I'm not 100% sure. Let's rerun it to check.

**Comment by bsmth** [comment]
> I think the failing test of Markdownlint is unrelated, but I'm not 100% sure. Let's rerun it to check.

Yeah I'm not 100% certain why it's failing. Needs a proper look 👀

**Comment by teoli2003** [comment]
I will try to add the prettier-induced changes to the other PR: that way, both the bump and the content changes will be in one single PR and there shouldn't be test failures (except the "System file changes" one).

I'm closing this one so we don't merge it by error. If it doesn't work, we can reopen it.

**Pull request closed without merging**